Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -56,9 +55,18 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         openSelectedPage.putExtra("content", content);
         openSelectedPage.putExtra("url", url);
         startActivity(openSelectedPage);
-        finish();
```

Comment:
Because I don't know about this function, is it okay to just remove this `finish()` method here?